player.py

```diff
@@ -19,19 +20,18 @@ class PlayerException(Exception):
     pass
 
 class Player():
-    def __init__(self, screen, map, position=(0, 0), size=(32, 32), colour=(255, 255, 255), name = "Name"):
+    def __init__(self, screen, map, colour=(255, 255, 255)):
         self.screen = screen
         self.map = map
         self.ready = False
         self.is_centre = False
-        self.size = size
-        self.step = self.size[0]
+        self.size = (TILE_PIX_WIDTH, TILE_PIX_HEIGHT)
+        self.step = 1
         self.colour = colour
-        self.name = name
+        self.name = 'Name'
 
-        if len(position) > 0:
```

Comment:
As of present, the only ill-effect should be that the player momentarily appears at `( 0, 0 )` until their position is updated.